src/adaptation/Adaptation.jl

```diff
@@ -1,75 +1,59 @@
 module Adaptation
+export Adaptation
```

Comment:
IMO it would simplify the import and export statements if the adaptation implementation is not in a separate module.

Reply:
I agree. Originally I was trying to implement these adaptations s.t. they might be used for another library in the future. However, we are moving towards the opposite direction - making adaption more tied to AdvancedHMC.jl now. So I think it's fair to not make them as a seperate module any more.

Reply:
Let's keep it as a module for now then.